pallets/dispatcher/src/benchmarking.rs

```diff
@@ -53,5 +53,14 @@ benchmarks! {
 	note_aave_manager {
 	}: _(RawOrigin::Root, Pallet::<T>::aave_manager_account())
 
+	dispatch_with_extra_gas{
+		let n in 1 .. 10_000;
+		let remark = sp_std::vec![1u8; n as usize];
+
+		let call: <T as pallet::Config>::RuntimeCall = frame_system::Call::remark { remark }.into();
```

Comment:
what is the reason that we benchmark it with just a remark call?

I would comment that reason in the code.

Reply:
we dont want to benchmark any specific call, because it would be included the actual weight.

remark does nothing. 

 the call weight is calculated separately for each call that you want to dispatch. and added to weight